engine/hid/src/hid.h

```diff
@@ -27,8 +27,8 @@
 
 namespace dmPlatform
 {
-    struct Window;
-    typedef Window* HWindow;
+    struct dmWindow;
+    typedef dmWindow* HWindow;
```

Comment:
"Window" is also a linux struct, so to avoid confusion I renamed our window to dmWindow